_data/tools.yml

```diff
@@ -158,6 +158,17 @@
   v2: true
   v3: true
 
+- name: BOATS
+  category: documentation
```

Comment:
Is this documentation? I know a lot of folks use OpenAPI for documentation, but this tool is helping folks write OpenAPI so if anything its almost like [a DSL](https://developer.squareup.com/blog/making-openapi-swagger-bearable-with-your-own-dsl/). Should we start a DSL category?

Reply:
Might be useful to have a DSL category.

Reply:
Cool, I update the PR. I would agree that boats is a kind of DSL, all be it a lighter version of the one you posted @philsturgeon